src/AccessibilityInsights.Desktop/UIAutomation/TreeWalkers/TreeWalkerForTest.cs

```diff
@@ -87,7 +87,8 @@ public void RefreshTreeData(TreeViewMode mode)
             // run tests
             list.AsParallel().ForAll(e =>
             {
-                SuiteFactory.RunRules(e);
+                e.ScanResults?.Items.Clear();
```

Comment:
this used to be inside `SuiteFactory.RunRules`. Since `SuiteFactory` is no longer needed, I've pulled out the one pre-processing step before calling the inner `RuleRunner`